.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Install packages
         run: |
           sudo apt update
-          sudo apt install -y libfuse-dev build-essential
+          sudo apt install -y libfuse-dev libfuse3-dev build-essential
```

Comment:
Do we need this in our ci container as well?

Reply:
If you build with libfuse (which is turned on by default), then yes. You have to build without default features to disable it